seonshine_frontend/src/utils/persistCache/index.ts

```diff
@@ -1,14 +1,19 @@
 class PersistCache {
-  static save(key: string, value: string): void {
-    localStorage.setItem(key, value);
+  static save(key: string, value: string, rememberMe?: boolean) {
```

Comment:
`persistCache` is just for save, read, remove, not for handling logic. RememberMe logic for authenticated must be handled from `auth.ts` file. BTW, PersistCache is just for Cookie or localStorage, not used for sessionStorage. If needed, use session storage. Please define `sessionCache`